src/main/java/com/seailz/discordjv/model/emoji/Emoji.java

```diff
@@ -30,7 +31,7 @@ public record Emoji(
         boolean managed,
         boolean animated,
         boolean available
-) implements Compilerable {
+) implements Compilerable, Mentionable {
```

Comment:
Please remove Mentionable from this, (it doesn't count!!).

Reply:
so do you want me to remove the mention code as well as the `Mentionable` implementation?

Reply:
No, just the implementation of the `Mentionable` class.